src/main/kotlin/org/rust/ide/newProject/ui/RsNewProjectPanel.kt

```diff
@@ -88,6 +89,7 @@ class RsNewProjectPanel(
 
     val templateToolbar: ToolbarDecorator = ToolbarDecorator.createDecorator(templateList)
         .setToolbarPosition(ActionToolbarPosition.BOTTOM)
+        .setPreferredSize(Dimension(0, 125))
```

Comment:
Let's use `com.intellij.util.ui.JBUI#size(int, int)` instead of raw `Dimension` to support IDE scaling when it's not performed by OS itself